src/credentials/FabricTable.h

```diff
@@ -385,6 +385,11 @@ class DLL_EXPORT FabricTable
          **/
         virtual void OnFabricCommitted(const FabricTable & fabricTable, FabricIndex fabricIndex){};
 
+        /**
+         * Gets called when a fabric is added to Fabric Table, by AddingNewPendingFabricCommon.
```

Comment:
Please remove implementation detail from comment.
```suggestion
         * Gets called when a fabric is newly added to Fabric Table.
```

Reply:
Merged the two functions